src/main/java/uk/gov/ons/ctp/response/action/service/ActionRequestDecorator.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.ons.ctp.response.action.service;
+
+import uk.gov.ons.ctp.response.action.message.instruction.ActionRequest;
+import uk.gov.ons.ctp.response.action.service.impl.decorator.ActionRequestDecoratorContext;
+
+public interface ActionRequestDecorator {
```

Comment:
The name Decorator is ambiguous. Does it mean the Decorator pattern or something that decorates the ActionRequest object? Maybe having a name `ActionRequestPopulator`, `ActionRequestEnricher`, `ActionRequestLoader` may give more value